cajita/unit_test/tstHalo3d.hpp

```diff
@@ -73,16 +70,9 @@ void checkGather( const int halo_width, const Array& array )
                          j >= owned_space.max( Dim::J ) + halo_width + pad_j ||
                          k < owned_space.min( Dim::K ) - halo_width ||
                          k >= owned_space.max( Dim::K ) + halo_width + pad_k )
-                    {
-                        if ( host_view( i, j, k, l ) != 0.0 )
-                            pass_gather_test = 0;
-                    }
+                        EXPECT_EQ( host_view( i, j, k, l ), 0.0 );
```

Comment:
This is better, but will take much longer until we fix the failing distributions

Reply:
OK I think these changes to this part of the test probably overwrote your updates when I did the rebase. I am going to revert back to your test structure for now.